nixos/tests/musescore.nix

```diff
@@ -0,0 +1,36 @@
+import ./make-test-python.nix ({ pkgs, ...} :
+
+{
+  name = "musescore";
+  meta = with pkgs.lib.maintainers; {
+    maintainers = [ turion ];
+  };
+
+  machine = { ... }:
+
+  {
+    imports = [
+      ./common/user-account.nix
+      ./common/x11.nix
+    ];
+
+    services.xserver.enable = true;
+    test-support.displayManager.auto.user = "alice";
+    environment.systemPackages = [ pkgs.musescore ];
+  };
+
+  enableOCR = true;
+
+  testScript = { nodes, ... }: let
+    user = nodes.machine.config.users.users.alice;
+  in ''
+    start_all()
+    machine.wait_for_x()
+
+    # start KeePassXC window
+    machine.execute("su - alice -c mscore &")
+
+    machine.wait_for_text("MuseScore")
```

Comment:
Would it be possible to actually do something in the app? Even just saving an empty file? This test could be satisfied by the main window opening without drawing and the DE drawing a "MuseScore is not responding" dialog.

It would be nice to at least input `<C-x>` and check that the target file exists. That we we know that input handling is working with is some test of the application booting.

Reply:
Looking into it! Is it ok to upload a fixture file (a midi export) against which I can diff so I know that the right content has been generated? Or is that too bloated (in terms of repo size)?